src/main/index.ts

```diff
@@ -145,20 +146,36 @@ if (!isFirstInstance) {
     mainWindow = newWindow;
   };
 
-  app.on('session-created', (session) => {
-    if (!isDevelopment) {
-      log.info('Cleaning local storage on session creation');
-      session.clearStorageData({ storages: ['localstorage'] }).catch(log.error);
-    }
-  });
-
   // This method will be called when Electron has finished
   // initialization and is ready to create browser windows.
   // Some APIs can only be used after this event occurs.
   app.on('ready', () => {
-    log.info('App ready');
+    log.info('App ready');
     createWindow();
 
+    if (isDevelopment) {
```

Comment:
The code inside this `if` statement will only run in development - you want this to run in production too, right?